### DISCUSSION TITLE
Arabic language

### DISCUSSION THREAD

**Discussion by asemandeveloper** [langgenius/dify#4847] [Suggestion]
Hello to all
I want to write the user interface for Arabic language and right-to-left display. Please guide me how to do this in the project code.

**Comment by patryk20120** [comment]
I don't know if RTL is supported right away in styles, if not you will have to make modifications to support it. Please check https://github.com/langgenius/dify/blob/main/web/i18n/README.md

**Reply by asemandeveloper** [reply]
Thank you very much for your guidance
I did what you said. How can I see the result in the project? What are the next steps?
